### DISCUSSION TITLE
Basel Deutsch

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#40] [General]
- Sbier isch im chüehlschrank ihgschlosse

**Comment by esthicodes** [comment]
- ich will kei Handcreme a mine Finger ha

**Comment by esthicodes** [comment]
- Eifach SRF2 live google!

**Comment by esthicodes** [comment]
- Das me nüt mit ihre sött zdue ha
- Dass man nichts mit ihr zu tun haben sollte.

**Comment by moodyauto** [comment]
Def. U & U' eines VR_s V mit den Eigenschaft dass jedes x / V ein eindeutige Darstellung x = u + u, u ist in u', hat heissen Komplementär(complimenting Subsequence)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/204089665-9ae6e30e-4d49-4230-9da2-0b5b1fc4e05a.jpeg)

**Comment by esthicodes** [comment]
Wenni frooge dörft

== Unterschiede Alltags- / älteres Baseldeutsch ==
Das heutige Alltags-Baseldeutsch unterscheidet sich im Wesentlichen wie folgt vom traditionellen Stadt-Baseldeutsch:
* Praktisch verschwunden ist die Entrundung, es gilt heute ''ö'' für ''e'', ''ü'' statt ''i'' und ''öi'' für ''ei''. Beispiele: «d'''ü'''rftig» statt «d'''i'''rftig», «Wenni frooge d'''ö'''rft» anstatt «Wenni frooge d'''e'''rft», «F'''üü'''r» [yː] (Feuer) statt «F'''yy'''r» [iː]; «n'''öi'''» [œi] (neu) statt «nei» [ei]; daher heisst es heute ''Baaseldütsch'' im Gegensatz zum traditionellen ''Baaseldytsch''. Nach wie vor weniger rückgerundet wird ''ie'' [iə], zum Beispiel «m'''ie'''se», seltener «m'''üe'''se» [yə] (müssen).
* Fortis ''k'' [k] oder ''ch'' [x] ersetzt die Lenis ''g'' im Anlaut, zum Beispiel «e '''k'''laineri Sach» oder «e '''ch'''laineri Sach» im Alltags-Baseldeutschen, «e '''g'''laineri Sach» im traditionellen Stadt-Baseldeutschen.
* ''r'' wird nicht mehr als ''ch'' [x] gesprochen, beispielsweise Alltags-Baseldütsch: «Bilde'''r'''» [ʀ]; traditionelles Stadt-Baseldeutsch: «Bild'''ch'''»
* Integration von ursprünglich Baseldeutsch-fremden Wörtern wie etwa «Grüezi» in Form von «Griezi» [gʀiətsɪ].

**Comment by esthicodes** [comment]
{{Infobox Sprache
|Sprache=Baseldeutsch
|Länder=[[Schweiz]] ([[Basel]])
|Sprecher=
|Klassifikation=
* [[Indogermanische Sprachfamilie|Indogermanisch]]
*: [[Germanische Sprachen|Germanisch]]
*:: [[Westgermanische Sprachen|West-Germanisch]]
*::: [[Deutsche Sprache|Deutsch]]
*:::: [[Oberdeutsche Sprachen|Oberdeutsch]]
*::::: [[Alemannische Dialekte|Alemannisch]]
*:::::: [[Niederalemannisch]]
|KSprache=Baseldeutsch
|Amtssprache=—
|ISO1=—
|ISO2=gsw (Schweizerdeutsch)
|ISO3=gsw (Schweizerdeutsch)
}}
'''Baseldeutsch''' (auch ''Baaseldytsch, Baseldytsch, Baseldütsch'') ist der Dialekt der Schweizer Stadt [[Basel]] und Umgebung. Es ist der einzige [[Schweizerdeutsch|deutschschweizerische Dialekt]], der dem [[Niederalemannisch]]en zugeordnet werden kann oder doch zumindest in seiner klassischen Form eine Reihe von Besonderheiten hat, die auch das Niederalemannische hat und die vom [[Hochalemannisch]]en abweichen.

**Comment by esthicodes** [comment]
== Typische Merkmale des traditionellen Stadt-Baseldeutsch ==
* Vokaldehnung in offener Silbe, d.&nbsp;h. vor einfachem Konsonanten ([ba:səl]; sonst hochalemannisch verbreitet [basəl])
* Kürzung von historischem Langvokal vor Fortiskonsonant unter Beibehaltung von dessen geschlossener Qualität, z.&nbsp;B. ''Lyt, dytsch, bysse, schlyffe'' (‚Leute, deutsch, beissen, schleifen‘) mit kurzem /i/ ([lit], [ditʃ], [bisːə], [ʃlifːə]).
* Entrundung von [[mittelhochdeutsch]] = hochalemannisch /y/ und /ø/ (''sch'''ee'''n'' [ʃ'''eː'''n], sonst hochalemannisch ''sch'''öö'''n'' [ʃ'''øː'''n ʃ'''œː'''n] ‚schön‘; ''gr'''i'''en'' [gʁ'''i'''ən], sonst hochalemannisch ''gr'''ü'''en'' [gr'''y'''ən] ‚grün‘); daher wird es auch ''Baaseldytsch'' genannt im Unterschied zum modernen ''Baaseldütsch''
* langes /u:/ wird zentralisiert artikuliert ([ʉ:mə] ‚herum‘)
* unverschobenes aspiriertes [kʰ] ([kʰind] ‚Kind‘; sonst hochalemannisch: [xind], sprich «chind»), auch aspirierte [tʰ, pʰ] sind etwas verbreiteter als in anderen Mundarten
* Lenisierung: Verschlusslaute im Wortanlaut werden meist lenisiert (also als stimmloses, kurzes [d]): ''Dyybli'' [ˈdiːblɪ] ‚Täubchen‘ (wie auch in gewissen anderen hoch- und niederalemannischen Dialekten)
* /r/ wird in den meisten Stellungen nicht als «Zäpfchen-''r''», sondern als [χ] = ''ch'' ausgesprochen ([ʋæːχ] ‚wer‘)
* das [[Suffix]] ''-lich'' wird als ''-lig'' realisiert (''meeglig'' [ˈmeːglɪg] ‚möglich‘; sonst hochalemannisch: ''mögli'' [ˈmøglɪ], ''möglech'' [ˈmœgləx])
* das Suffix der Zahlwörter von ''20'' bis ''90'' wird wie in der Schriftsprache realisiert ([ˈdʁisːɪg] ''30'', [ˈfuftsɪg] ''50;'' sonst im Hochalemannischen (ausgenommen Bündner und z.&nbsp;T. St.&nbsp;Galler Rheintal) ''driisg'' [ˈdriːsːg], ''füftsg'' [fʏftsg])
* Das Zahlwort ''1000'' wird mit Diphthong ausgesprochen ([ˈdausɪg]; sonst [ˈtuːsɪg ˈduːsɪg]).

**Comment by esthicodes** [comment]
== Sprachgebrauch des traditionellen Stadt-Baseldeutsch ==
Das alte Stadt-Baseldeutsch ''(Baaseldytsch)'' mit all seinen charakteristischen Merkmalen und Unterschieden zum heutigen Alltags-Baseldeutsch wird heute nur noch von einem kleinen, meist älteren Teil der Basler Bevölkerung gesprochen; jedoch sind praktisch alle Baslerinnen und Basler sehr wohl in der Lage, das alte Baseldeutsch zu verstehen oder gar nachzuahmen.

Allerdings war der Sprachgebrauch schon immer auch abhängig von der sozialen Schicht. Das, was heute als «altes, korrektes» Baseldeutsch gehandelt wird, entspricht dem Dialekt, wie er um das Ende des 19. Jahrhunderts in den gehobeneren Kreisen (dem ''[[Daig]]'') gesprochen wurde. In den sozial niedrigeren Kreisen (z.&nbsp;B. «Rheingasse») wurde es nie in dieser Form gesprochen.

**Comment by esthicodes** [comment]
== Das heutige Baseldeutsch ==
Heute dominiert das Alltags-Baseldeutsch, welches nicht nur die Stadt, sondern auch die nähere Umgebung umfasst. Das Alltags-Baseldeutsch ist das Produkt einer anhaltenden Annäherung zwischen dem traditionellen Stadt-Baseldeutsch und den baseldeutschen Dialekten im [[Kanton Basel-Landschaft|Baselbiet]], [[Fricktal]], [[Laufental]] und [[Schwarzbubenland]], was auf die massive Zuwanderung Ende des 19.&nbsp;Jahrhunderts und Anfang des 20.&nbsp;Jahrhunderts zurückgeht. Die Anpassung ist gegenseitig: Während in der Stadt die Entrundung (''ee'' [eː] für ''öö'' [øː], ''è'' [ɛ] für ''ö̀'' [œ], ''yy'' [iː] für ''üü'' [yː], ''ì'' [ɪ] für ''ü'' [ʏ], ''ei'' [ei] für ''öi'' [øi]) in den meisten Fällen aus der Alltagssprache verschwunden ist, breiten sich typische Stadt-Baseldeutsche Merkmale wie das dominante «Zäpfchen-r» über die Stadtgrenzen in die Agglomeration aus. Das heutige Baseldeutsch (Baaseldütsch) steht dem [[Hochalemannisch]]en näher als das ältere Baseldeutsch (Baaseldytsch). Dennoch haben sich einzelne typische Merkmale der Stadtsprache wie entrundetes ''ie'' [iə] für ''üe'' [yə] und anlautendes [k] bzw. [g] für hochalemannisches ''ch'' [χ] bis heute im baselstädtischen Dialekt erhalten. Die Dialekte im Baselbiet, Fricktal, Laufental und Schwarzbubenland gehören hingegen seit jeher zum Hochalemannischen.

**Comment by esthicodes** [comment]
== Wird das traditionelle Stadt-Baseldeutsch verdrängt? ==
Von einem Verschwinden des traditionellen Stadt-Baseldeutschen kann trotz der Entwicklungen nicht die Rede sein. Baslerinnen und Basler gehen selbstbewusst mit ihrer Sprache um. So ist die schriftliche Form des traditionellen Stadt-Baseldeutsch praktisch an jedem Ort der Stadt anzutreffen: Auf Speisekarten, Werbetafeln oder in Zeitungen. Es existieren auch ein Baseldeutsch-Wörterbuch und eine Grammatik. Eine besondere Bedeutung hat das traditionelle Stadt-Baseldeutsch in der [[Basler Fasnacht]], wo darauf geachtet wird, möglichst nicht vom «reinen» Baseldeutsch abzuweichen.

Das Alltagsbaseldeutsch bildet die mündliche Umgangssprache und wird beinahe ausschliesslich auch im lokalen Fernsehen und den Radiostationen verwendet.

**Comment by esthicodes** [comment]
== Die Basler Interessengemeinschaft Dialekt (IG Dialekt) ==
Zu Empörung bei Teilen der Öffentlichkeit hat 2008 die Ankündigung des Erziehungsdepartementes geführt, dass im Vorschulbereich Kindergärtnerinnen bis auf sogenannte «Dialektfenster» nur noch Standarddeutsch verwenden dürfen. Die Umsetzung der Anordnung wurde zurückgenommen. In der Folge bildete sich eine «Basler Interessengemeinschaft Dialekt» (IG Dialekt), der u.&nbsp;a. Dialektdichter [[Carl Miville-Seiler|Carl Miville]] und [[Rudolf Suter (Germanist)|Rudolf Suter]], Autor der Baseldeutsch-Standardwerke, angehören. Im Vorstand der IG Dialekt sind Personen aus sämtlichen Parteien vertreten. Ziele der Interessengemeinschaft sind: Dialekt als Umgangssprache im Kindergarten (mit 20 Prozent Hochdeutschfenster); Standarddeutsch als Unterrichtssprache an der Schule und Dialekt als Umgangssprache ausserhalb des Unterrichts.

Die IG Dialekt lancierte eine stadtbaslerische Volksinitiative, die verlangt: «Die Unterrichtssprache in den ersten beiden Jahren nach der Einschulung (Kindergartenstufe) ist Dialekt. Hochdeutsch wird in definierten Sequenzen gefördert.» Innert weniger Wochen wurden über 5000 Unterschriften gesammelt. Nötig wären 3000 in einer Sammelfrist von anderthalb Jahren.

Trotz der hängigen Initiative wurde auf das Schuljahr 2009/2010 für den Kindergarten die Regelung erlassen, dass von der Kindergärtnerinnen und Kindergärtnern mindestens 50 Prozent Standarddeutsch verwendet werden muss. Zuvor wurde – mit Ausnahme der sogenannten Versuchskindergärten – Mundart als Unterrichtssprache verwendet.

Über die Initiative wurde am 15. Mai 2011 abgestimmt, wobei die Regierung einen Gegenvorschlag zur Initiative vorlegte. Die Initiative wurde mit 55,11 % Ja-Stimmen angenommen, der Gegenvorschlag mit 51,86 % Ja-Stimmen. Da in der Stichfrage 222 Stimmen mehr auf den Gegenvorschlag als auf die Initiative entfielen, wurde somit der Gegenvorschlag angenommen.<ref>{{Internetquelle |url=https://www.bzbasel.ch/basel/basel-stadt/dialekt-initiative-knapp-abgelehnt-ja-zu-gegenvorschlag-108261316 |titel=Dialekt-Initiative knapp abgelehnt - Ja zu Gegenvorschlag |datum=2011-05-15 |abruf=2019-05-05}}</ref>

Ähnliche Volksinitiativen gab es etwa in den Kantonen Zürich und Luzern.

**Comment by esthicodes** [comment]
== Baseldeutsch im Schweizer Film ==
Im vor allem in [[Zürich]] produzierten Schweizer Dialektfilm diente Baseldeutsch lange zur Markierung des Bösewichts. Dieser Stereotyp ist schon im ältesten erhaltenen Dialektfilm ''[[Jä-soo!]]'' aus dem Jahr 1935 ersichtlich und kommt darauf in den ''[[Die missbrauchten Liebesbriefe|Missbrauchten Liebesbriefen]]'' (1940), dem im Zuge der [[Geistige Landesverteidigung|Geistigen Landesverteidigung]] entstandenen ''[[Landammann Stauffacher]]'' (1941), der Gotthelf-Verfilmung ''[[Uli der Pächter (Film)|Uli der Pächter]]'' (1955) und dem in der Stadt Zürich spielenden ''[[Bäckerei Zürrer]]'' (1957) zum Tragen. Über Dokumentarfilme und [[Dani Levy]]s «Peperoni» in der Fernsehserie ''[[Motel (Schweizer Fernsehserie)|Motel]]'' aus den 1980ern konnte sich der Basler Dialekt filmisch rehabilitieren.<ref name="aeppli">[http://aeppli.ch/Works/Baseldeutsch.pdf aeppli.ch] (PDF; 24&nbsp;kB): Felix Aeppli: ''Vorsicht Baseldeutsch! Zur Funktion des Dialekts im Schweizer Film.'' Aus ''Zürcher Filmrollen'' (hrsg. von der [[Zürcher Kantonalbank]]), Zürich 2005.</ref>

**Comment by esthicodes** [comment]
== Literatur ==
'''Wörterbücher'''
* [[Christoph Merian Stiftung]] (Hrsg.): ''Neues Baseldeutsch-Wörterbuch.'' Christoph Merian Verlag, Basel 2010, ISBN 978-3-85616-502-4.
* [[Rudolf Suter (Germanist)|Rudolf Suter]]: ''Baseldeutsch-Wörterbuch.'' 3.&nbsp;Auflage. Christoph Merian Verlag, Basel 2006 (Grammatiken und Wörterbücher zum Schweizerdeutschen V), ISBN 978-3-85616-305-1.
* ''Fridolin'' [= [[Robert Balthasar Christ|Robert B. Christ]]]. ''E Baseldytsch-Sammlig. Ygruumt in zwelf Fächli und in e Vytryne. Mit Helge vom Ferdi Afflerbach.'' Birkhäuser, 4.&nbsp;Auflage Basel 1976 (Grammatiken und Wörterbücher zum Schweizerdeutschen V).
* [[Gustav Adolf Seiler (Philologe)|Gustav Adolf Seiler]]: ''Die Basler Mundart. Ein grammatisch-lexikographischer Beitrag zum schweizerdeutschen Idiotikon, zugleich ein Wörterbuch für Schule und Haus.'' Detloff, Basel 1879; unveränderter Nachdruck: Sändig Reprint, Wiesbaden 1970.
* [[Johann Jacob Spreng]]: ''Idioticon Rauracum oder Baselisches Wörterbuch.'' Manuskript um 1760. Hrsg. von [[Heinrich Löffler (Germanist)|Heinrich Löffler]] unter dem Titel ''Idioticon Rauracum oder Baseldeutsches Wörterbuch von 1768. Edition der Handschrift AA&nbsp;I&nbsp;3 der Universitätsbibliothek Basel.'' Schwabe, Basel 2014.

'''Grammatiken'''
* Rudolf Suter: ''Baseldeutsch-Grammatik.'' 3. Auflage. Christoph Merian Verlag, Basel 1992 (Grammatiken und Wörterbücher zum Schweizerdeutschen VI), ISBN 3-85616-048-5.
* [[Herbert Pilch]]: ''Baseldeutsche Phonologie. Auf Grundlage der Intonation.'' In: ''Phonetica'' 34 (1977), S.&nbsp;165–190.
* [[Eduard Hoffmann-Krayer|Eduard Hoffmann]]: ''Der mundartliche Vokalismus von Basel-Stadt in seinen Grundzügen dargestellt.'' Adolf Geering’s, Basel 1890.
* [[Andreas Heusler (Altgermanist)|Andreas Heusler]]: ''Der alemannische Consonantismus in der Mundart von Baselstadt.'' Dissertation Basel. Karl J. Trübner, Strassburg 1888.
* Gustav Binz: ''Zur Syntax der baselstädtischen Mundart.'' Inaugural-Dissertation Basel. Kröner, Stuttgart 1888.

'''Geschichte und Entwicklung'''
* Ernst Erhard Müller: ''Die Basler Mundart im ausgehenden Mittelalter.'' Francke, Bern 1953 (Basler Studien zur deutschen Sprache und Literatur 14).
* [[Adolf Socin]]: ''Zur Geschichte der Basler Mundart.'' In: [[Franz August Stocker]]: ''Vom Jura zum Schwarzwald. Geschichte, Sage, Land und Leute.'' Aarau 1888, S.&nbsp;81–95.
* [[Wilhelm Bruckner]]: ''Veränderungen im Wortschatz der Basler Mundart.'' In: ''Teuthonista'' 8 (1931/1932), S.&nbsp;170–197.
* Albert Gessler: ''Beiträge zur Geschichte der Entwicklung der neuhochdeutschen Schriftsprache in Basel.'' Inaugural-Dissertation Basel. Frehner & Rudin, Basel 1888.
* Rudolf Suter: ''Die baseldeutsche Dichtung vor J.&nbsp;P.&nbsp;Hebel. Basler Mundart und Mundartforschung im 17. und 18.&nbsp;Jahrhundert.'' Dissertation Basel. Vineta, Basel 1949.
* Eduard Hoffmann-Krayer: ''Werden und Wandeln der Basler Mundart.'' In: ''Sonntagsblatt der «Basler Nachrichten»,'' 15.&nbsp;Jahrgang, Juni/Juli 1921, Nummer 25, 26, 27. Auch als durchpaginierter Separatabzug gedruckt.
* [[Wilhelm Altwegg]]: ''Baseldytsch.'' In: ''Basel, ein Stadtbuch.'' Basel [1932].
* [[Robert Balthasar Christ|Robert B. Christ]]: ''U und Non-U in Basel. Standesunterschiede in der Basler Mundart.'' In: ''Sprachspiegel'' 19 (1963), S.&nbsp;129–135 ([[doi:10.5169/seals-420726]]).
* Lorenz Hofer: ''Sprachwandel im städtischen Dialektrepertoire. Eine variationslinguistische Untersuchung am Beispiel des Baseldeutschen.'' Francke, Basel/Tübingen 1997 (Basler Studien zur deutschen Sprache und Literatur 72).
* Beatrice Bürki: ''Sprachvariation in einem Grossbetrieb. Eine individuenzentrierte Analyse anhand sprachlicher Tagesläufe.'' Francke, Basel/Tübingen 1999 (Basler Studien zur deutschen Sprache und Literatur 73).
* Petra Leuenberger: ''Ortsloyalität als verhaltens- und sprachsteuernder Faktor. Eine empirische Untersuchung.'' Francke, Basel/Tübingen 2000 (Basler Studien zur deutschen Sprache und Literatur 74).
* Lorenz Hofer, mit Beiträgen von [[Annelies Häcki Buhofer]] und Heinrich Löffler: ''Zur Dynamik urbanen Sprechens. Studien zu Spracheinstellungen und Dialektvariation im Stadtraum.'' Francke, Basel/Tübingen 2002 (Basler Studien zur deutschen Sprache und Literatur 71).

'''Baseldeutsch im alemannischen Zusammenhang'''
* ''[[Schweizerisches Idiotikon]],'' Bände I&nbsp;ff. Huber, Frauenfeld 1881–2012 bzw. Schwabe, Basel 2014&nbsp;ff.
* ''[[Sprachatlas der deutschen Schweiz]],'' Bände I–VIII. Francke, Bern, später Basel 1962–1997.
* [[Rudolf Hotzenköcherle]]: ''Der Nordwesten.'' In: Rudolf Hotzenköcherle: ''Die Sprachlandschaften der deutschen Schweiz.'' Hrsg. von Niklaus Bigler und Robert Schläpfer unter Mitarbeit von Rolf Börlin. Aarau / Frankfurt a.&nbsp;M. / Salzburg 1984 (Reihe Sprachlandschaft 1), S.&nbsp;71–77.

'''Belletristik''' (Anthologie)
* Rudolf Suter (Hrsg.): ''Uff baaseldytsch. 100 baaseldytschi Täggscht us 200 Joor.'' Friedrich Reinhardt, Basel 1988.

**Comment by esthicodes** [comment]
== Einzelnachweise ==
<references />

{{Normdaten|TYP=s|GND=4120154-1}}

[[Kategorie:Schweizerdeutsch]]
[[Kategorie:Alemannischer Dialekt]]
[[Kategorie:Kultur (Basel)]]

**Comment by Esrhicodes** [comment]
>  zum Glück gibts noch mehr Gerber Biscuits
Zum Glück gits no meh Gärber Biscuits
